core/jazz_logs/config/dev-config.json

```diff
@@ -1,47 +1,47 @@
 {
-	"BASE_URL" :"https://{inst_elastic_search_hostname}/",
-	"DEFAULT_SIZE" : 1000,
-	"DEFAULT_TIME_IN_DAYS" : 7,
-	"VALID_ENVIRONMENTS" : ["prod", "stg", "dev"],
-	"VALID_CATEGORIES" : ["api", "function"],
-	"VALID_LOGTYPES" : ["warn", "error", "info", "verbose", "debug", "fatal", "trace", "off", "all"],
-	"ENV_PREFIX" : "{env-prefix}",
-	"LOG_LEVELS": [
-        {
-            "Level": 0,
-            "Type": "off"
-        },
-        {
-            "Level": 1,
-            "Type": "fatal"
-        },
-        {
-            "Level": 2,
-            "Type": "error"
-        },
-        {
-            "Level": 3,
-            "Type": "warn"
-        },
-        {
-            "Level": 4,
-            "Type": "info"
-        },
-        {
-            "Level": 5,
-            "Type": "debug"
-        },
-        {
-            "Level": 6,
-            "Type": "trace"
-        },
-        {
-            "Level": 7,
-            "Type": "verbose"
-        },
-        {
-            "Level": 8,
-            "Type": "all"
-        }
-    ]
-}
\ No newline at end of file
+  "BASE_URL": "https://{inst_elastic_search_hostname}/",
+  "DEFAULT_SIZE": 1000,
+  "DEFAULT_TIME_IN_DAYS": 7,
+  "VALID_ENVIRONMENTS": ["prod", "stg", "dev"],
+  "VALID_CATEGORIES": ["api", "function"],
+  "VALID_ASSET_TYPES": ["apilogs", "applicationlogs"],
```

Comment:
I think asset_types should be similar to what we use at other places. These are valid asset types - https://github.com/tmobile/jazz/blob/sls_app/core/jazz_assets/config/global-config.json#L25

And, there has to be a mapping b/w asset_type & log index that we use in ES. 
For ex: 
`asset_type: lambda -> es_indes: applicationlogs/functionlogs`
`asset_type: apigateway -> es_indes: apilogs`

Reply:
Included list of asset types and asset-index mapping as mentioned.

Reply:
changes look good.